CSharpAdvanced/GenericHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+public static class GenericHelper
```

Comment:
please change naming. GenericHelper? What it can help with? Costs? Chamomiles? Flowers?